### PR TITLE
PlaneFight runway Length 

### PR DESCRIPTION
**Pull request by Wood-VCW3**
Completed fighter minimum runway length distance. 

Note: I noticed that all vintage prop aircraft have a max cruise altitude of 15000' this should be at least 20000 to 25000

### CODE REVIEW

File: Database/Units/PlaneFighter/MiG-29S.ini [hunk 1]
```diff
@@ -15,6 +15,7 @@ CarrierTypes=
 
 CruiseAltitude=35000
 CruiseSpeed=500
+MinimumRunwayLengthFt=787
```
Comment (john681611) [inline Database/Units/PlaneFighter/MiG-29S.ini:18]: Seems a little short

**Review by john681611 — COMMENTED** [review]
Just double-checking some values that may be incorrect as they seem very short

**Comment by VCW3-admin** [comment]
That’s what I thought as well but checked a few other sites and came with the same number. That’s the F15 at full performance interception takeoff. Same thing with the Mig 29 and if I’m not mistaken. The hard ones I could not find was the vintage so they are based on the ME109 with similarities. I’ll post the references for the eagle and 29.

**Comment by john681611** [comment]
Fair enough as we said we will adjust as nessasarry

**Comment by Wood-VCW3** [comment]
Those are the numbers. however if you think they are to low you can always increase them to landing distance since that is a minimum distance as well. At time Takeoff distance is governed by landing distance as well. I've edited the F-15 and F-15E to use the minimum landing distance which is still very impressive.

**Pull request merged**